### PR TITLE
Done Open source 

### PR DESCRIPTION
**Pull request by NicolaePaulGeambazu**

### CODE REVIEW

File: src/_asset/font/dist/style.css [hunk 1]
```diff
@@ -0,0 +1,128 @@
+@charset "UTF-8";
+@font-face {
+  font-family: "my-cv-junior";
+  src: url("asset/font/my-cv-junior.ttf?ilb2j7") format("truetype"), url("asset/font/my-cv-junior.woff?ilb2j7") format("woff"), url("asset/font/my-cv-junior.svg?ilb2j7#my-cv-junior") format("svg");
+  font-weight: normal;
+  font-style: normal;
+  font-display: block;
+}
```
Comment (MariaTeo) [inline src/_asset/font/dist/style.css:8]: You could put an empty line for readability ^^

File: src/_asset/dist/style.css [hunk 2]
```diff
@@ -0,0 +1,128 @@
+@charset "UTF-8";
+@font-face {
+  font-family: "my-cv-junior";
+  src: url("asset/font/my-cv-junior.ttf?ilb2j7") format("truetype"), url("asset/font/my-cv-junior.woff?ilb2j7") format("woff"), url("asset/font/my-cv-junior.svg?ilb2j7#my-cv-junior") format("svg");
+  font-weight: normal;
+  font-style: normal;
+  font-display: block;
+}
```
Comment (MariaTeo) [inline src/_asset/dist/style.css:8]: You should put an empty line for readability ^^

File: src/_asset/font/dist/style1.css [hunk 3]
```diff
@@ -0,0 +1,128 @@
+@charset "UTF-8";
+@font-face {
+  font-family: "my-cv-junior";
+  src: url("asset/font/my-cv-junior.ttf?ilb2j7") format("truetype"), url("asset/font/my-cv-junior.woff?ilb2j7") format("woff"), url("asset/font/my-cv-junior.svg?ilb2j7#my-cv-junior") format("svg");
+  font-weight: normal;
+  font-style: normal;
+  font-display: block;
+}
```
Comment (MariaTeo) [inline src/_asset/font/dist/style1.css:8]: You could put an empty line for readability ^^

File: src/_asset/fonts/dist/style.css [hunk 4]
```diff
@@ -0,0 +1,128 @@
+@charset "UTF-8";
+@font-face {
+  font-family: "my-cv-junior";
+  src: url("fonts/my-cv-junior.ttf?ilb2j7") format("truetype"), url("fonts/my-cv-junior.woff?ilb2j7") format("woff"), url("fonts/my-cv-junior.svg?ilb2j7#my-cv-junior") format("svg");
+  font-weight: normal;
+  font-style: normal;
+  font-display: block;
+}
```
Comment (MariaTeo) [inline src/_asset/fonts/dist/style.css:8]: You could put an empty line for readability ^^

File: src/_component/experience/experience.scss [hunk 5]
```diff
@@ -1,3 +1,95 @@
...
+  &:checked ~ .experience__chevron , .icon__arrow-up {
+     transform: rotate(0);
+  }
+}
+
+.experience__date {
+   font-style: italic;
+   font-size: 12px;
+   line-height: 30px;
+}
```
Comment (MariaTeo) [inline src/_component/experience/experience.scss:91]: You could put an empty line for readability ^^

File: src/dist/style.css [hunk 6]
```diff
@@ -505,17 +508,118 @@ a.button__download--albastrel {
   margin-right: 10px;
 }
 
+.experience__item {
+  display: grid;
+  grid-template-columns: auto 1fr auto;
+}
```
Comment (MariaTeo) [inline src/dist/style.css:569]: You could put an empty line for readability ^^

File: src/dist/style.css [hunk 7]
```diff
@@ -505,17 +508,118 @@ a.button__download--albastrel {
...
+  margin: 25px;
+  width: 40px;
+  height: 40px;
+  border: 2px solid #707070;
+  border-radius: 50%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  cursor: pointer;
+}
```
Comment (MariaTeo) [inline src/dist/style.css:626]: You could put an empty line for readability ^^

File: src/dist/style.css [hunk 8]
```diff
@@ -505,17 +508,118 @@ a.button__download--albastrel {
...
+.experience__chevron .icon__arrow-up {
+  color: black;
+  transform: rotate(180deg);
+}
+
+.experience__checkbox {
+  display: none;
+}
```
Comment (MariaTeo) [inline src/dist/style.css:634]: You could put an empty line for readability ^^

File: src/dist/style.css [hunk 9]
```diff
@@ -505,17 +508,118 @@ a.button__download--albastrel {
...
+.experience__checkbox:checked ~ .experience__body {
+  display: block;
+}
```
Comment (MariaTeo) [inline src/dist/style.css:637]: You could put an empty line for readability ^^

File: src/dist/style.css [hunk 10]
```diff
@@ -505,17 +508,118 @@ a.button__download--albastrel {
...
+
 @font-face {
   font-family: "my-cv-junior";
-  src: url("fonts/my-cv-junior.ttf?ilb2j7") format("truetype"), url("fonts/my-cv-junior.woff?ilb2j7") format("woff"), url("fonts/my-cv-junior.svg?ilb2j7#my-cv-junior") format("svg");
+  src: url("asset/font/my-cv-junior.ttf?ilb2j7") format("truetype"), url("asset/font/my-cv-junior.woff?ilb2j7") format("woff"), url("asset/font/my-cv-junior.svg?ilb2j7#my-cv-junior") format("svg");
   font-weight: normal;
   font-style: normal;
   font-display: block;
 }
```
Comment (MariaTeo) [inline src/dist/style.css:659]: You could put an empty line for readability ^^

**Review by MariaTeo — COMMENTED** [review]
great WIP

**Review by M00N89 — COMMENTED** [review]
ok

**Pull request closed without merging**